### PR TITLE
Navigation

### LINKED ISSUE #14: Add Navigation
## Value statement ## 
As a **user**
I need **a navigation**
so **I can switch between the contents that are available**

## Description ##
Bildschirmfoto 2020-09-08 um 11 19 37

## Acceptance criteria ##
- [x] Static Navigation on the bottom of the page
- [x] Navigation must be shown on all pages
- [x] 3 Buttons available 
- [x] Buttons switch to the according pages on click

## Task ##

- [x] Create Navigation component
- [x] Find svgs for buttons
- [x] Install React Router Dom
- [x] Unit Test
- [x] Write Story
**(Create pages later)**

## Complexity ##
small

### PR DESCRIPTION
**Pull request by denizulku**
Please review :) 
I would like to have feedback on my App.test.js also.

### CODE REVIEW

File: src/components/Navigation/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  return (
+    <NavigationStyled>
+      <NavLink exact to="/">
+        <img src={IconHome} alt="Home" />
+      </NavLink>
+      <NavLink to="/Create">
+        <AddIcon src={IconPlus} alt="Create" />
+      </NavLink>
+      <NavLink to="/Overview">
+        <img src={IconMonthlyOverview} alt="Monthly Overview" />
```
Comment (carolinwagner) [inline src/components/Navigation/Navigation.js]: Your alt attributes are a little imprecise - If a user, who needs a screen-reader will hear the description "home" I guess it is not very helpful. Maybe you can precise it a bit more or even remove it if it has no value for such users.

File: src/components/Navigation/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+
+const NavigationStyled = styled.div`
+  font-size: 100%;
+  height: 64px;
+  width: 100%;
+  border-top-right-radius: 10px;
+  border-top-left-radius: 10px;
+  position: fixed;
+  bottom: 0px;
```
Comment (carolinwagner) [inline src/components/Navigation/Navigation.js]:
```suggestion
  bottom: 0;
```

File: src/components/Navigation/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+  background: #bcbbbc;
```
Comment (carolinwagner) [inline src/components/Navigation/Navigation.js:38]: Maybe you can use the CSS variables to increase readability

**Review by carolinwagner — COMMENTED** [review]
Nice work! I just left some minor comments :-)

File: src/components/Navigation/Navigation.js (hunk 3 above)
Comment (denizulku) [reply src/components/Navigation/Navigation.js:38]: I only use this color once for the nav, but thanks for the tip, will do for the other styles! :)

**Pull request merged**